SU-HKKU/cardinal_service_profile
# CSD-233 Created spotlight list view and filters

# READY FOR REVIEW

# Summary
- A bunch of changes for spotlights
- Consolidate graduation year and major into a text field
- Made resource plugin more general and accept different content types.
- Added field to spotlights to tag with person type (under grad, faculty etc)
- created a react filters block for spotlight stories
- A bunch of styling is coming next.

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. `composer require SU-HKKU/cardinal_service_profile:dev-CSD-233_spotlight-view --no-update; composer update -n`
1. `blt drupal:sync --site=cardinalservice`
1. Make sure the database updates and the config imports don't fail
1. all other testing will be done on the prod site with live content during QA.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
